integration_tests/tests/rekognition.rs

```diff
@@ -0,0 +1,17 @@
+#![cfg(feature = "rekognition")]
+
+extern crate rusoto_core;
+extern crate rusoto_rekognition;
+
+use rusoto_rekognition::{Rekognition, RekognitionClient, ListCollectionsRequest};
+use rusoto_core::{DefaultCredentialsProvider, Region, default_tls_client};
+
+#[test]
+fn should_list_collections() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = RekognitionClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
+    let request = ListCollectionsRequest::default();
+
+    let result = client.list_collections(&request).unwrap();
+	println!("{:#?}", result);
```

Comment:
Those ones were fine, strangely. How about we just merge these and then run a `rustfmt` on the `integration_tests` project in its entirety after? There's a few other things that would be fixed up in existing tests with a rustfmt as well.